Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.cpp

```diff
@@ -57,6 +58,11 @@ ServiceWorkerDownloadTask::ServiceWorkerDownloadTask(NetworkSession& session, Ne
     , m_downloadID(downloadID)
     , m_networkProcess(serviceWorkerConnection.networkProcess())
 {
+    m_expectedContentLength = response.expectedContentLength();
```

Comment:
expectedContentLength() should probably be changed to return a std::optional<uint64_t>.  Its -1 value should be nullopt, but it was written before std::optional existed.  It shouldn't be stored as a std::optional, though, because that would increase the size of a common structure, ResourceResponseBase.

Reply:
Oh, I see why it is a int64_t now.
I can probably do this as a follow-up.